sql/executor.go

```diff
@@ -63,7 +65,10 @@ func newExecutor(db client.DB, gossip *gossip.Gossip, clock *hlc.Clock) *Executo
 
 // SetNodeID sets the node ID for the SQL server.
 func (e *Executor) SetNodeID(nodeID roachpb.NodeID) {
+	// This method is called during node setup before any requests are received
+	// making it safe to perform without locking.
```

Comment:
nit: that's making assumptions on the caller. Just add to the function comment that this must be called before actually using the `Executor`.